internal/internal_task_pollers.go

```diff
@@ -335,29 +333,30 @@ func (wtp *workflowTaskPoller) processResetStickinessTask(rst *resetStickinessTa
 	return nil
 }
 
-func (wtp *workflowTaskPoller) RespondTaskCompletedWithMetrics(completedRequest interface{}, taskErr error, task *workflowservice.PollWorkflowTaskQueueResponse, startTime time.Time) (response *workflowservice.RespondWorkflowTaskCompletedResponse, err error) {
+func (wtp *workflowTaskPoller) RespondTaskCompletedWithMetrics(
+	completedRequest interface{},
+	taskErr error,
+	task *workflowservice.PollWorkflowTaskQueueResponse,
+	startTime time.Time,
+) (response *workflowservice.RespondWorkflowTaskCompletedResponse, err error) {
 
 	if taskErr != nil {
-		wtp.metricsScope.Counter(metrics.WorkflowTaskExecutionFailedCounter).Inc(1)
+		wtp.metricsScope.Counter(metrics.WorkflowTaskExecutionFailureCounter).Inc(1)
 		wtp.logger.Warn("Failed to process workflow task.",
 			tagWorkflowType, task.WorkflowType.GetName(),
 			tagWorkflowID, task.WorkflowExecution.GetWorkflowId(),
 			tagRunID, task.WorkflowExecution.GetRunId(),
 			tagError, taskErr)
 		// convert err to WorkflowTaskFailed
 		completedRequest = errorToFailWorkflowTask(task.TaskToken, taskErr, wtp.identity, wtp.dataConverter)
-	} else {
-		wtp.metricsScope.Counter(metrics.WorkflowTaskCompletedCounter).Inc(1)
 	}
 
 	wtp.metricsScope.Timer(metrics.WorkflowTaskExecutionLatency).Record(time.Since(startTime))
 
-	responseStartTime := time.Now()
 	if response, err = wtp.RespondTaskCompleted(completedRequest, task); err != nil {
-		wtp.metricsScope.Counter(metrics.WorkflowTaskResponseFailedCounter).Inc(1)
+		wtp.metricsScope.Counter(metrics.WorkflowTaskNoCompletionCounter).Inc(1)
```

Comment:
I am not sure if this is a right usage of this metric.

Reply:
I'm not sure it is needed as RespondTaskCompleted counts errors anyway.

Reply:
Do we have it in Java?